ERCS/erc-7683.md

```diff
@@ -98,79 +111,240 @@ struct Input {
 struct Output {
 	/// @dev The address of the ERC20 token on the destination chain
 	/// @dev address(0) used as a sentinel for the native token
-	address token;
+	bytes32 token;
 	/// @dev The amount of the token to be sent
 	uint256 amount;
 	/// @dev The address to receive the output tokens
-	address recipient;
+	bytes32 recipient;
 	/// @dev The destination chain for this output
 	uint32 chainId;
 }
+```
+
+### Initiated event
+
+A compliant `Initiated` event MUST adhere to the following abi:
+
+```solidity
+/// @title FillInstructions type
+/// @notice Instructions to parameterize each leg of the fill
+/// @dev Provides all the origin-generated information required to produce a valid fill leg
+struct FillInstructions {
+	/// @dev The contract address that the order is meant to be settled by
+	uint32 destinationChainId;
+	/// @dev The contract address that the order is meant to be filled on
+	bytes32 destinationSettler;
+	/// @dev The data generated on the origin chain necessary to inform the fill
+	bytes originData;
```

Comment:
It was intended to differentiate from fillerData on the destination (since that is also used for settlement). Could also be called settlerData?

No strong opinions. Naming is hard :)